tests/failing_test.go

```diff
@@ -127,6 +128,19 @@ func TestDeserializeKE3(t *testing.T) {
 	}
 }
 
+func TestSetAKEState(t *testing.T) {
+	c := opaque.DefaultConfiguration()
+	macLength := c.MAC.Size()
+	keyLength := c.KDF.Size()
+
+	buf := internal.RandomBytes(macLength + keyLength + 1)
+
+	server := c.Server()
+	if err := server.SetAKEState(buf); err == nil || err.Error() != errInvalidStateLength.Error() {
+		t.Fatalf("Expected error for SetAKEState. want %q, got %q", errInvalidStateLength, err)
+	}
+}
```

Comment:
If there are more tests needed here, I'm happy to implement. From what I saw, it looked like this was the only one needed.

Reply:
Yes, it's just this test :) Thanks!